workbench-core/authorization/src/dynamicAuthorization/wbcGroupManagementPlugin.test.ts

```diff
@@ -65,6 +74,43 @@ describe('WBCGroupManagemntPlugin', () => {
     ddbMock.reset();
   });
 
+  describe('createGroup', () => {
+    it('returns `created` as true when the group was successfully created', async () => {
+      const groupId = 'groupId';
+
+      const response = await wbcGroupManagementPlugin.createGroup({ groupId, authenticatedUser: mockUser });
+
+      expect(response).toMatchObject<CreateGroupResponse>({ data: { groupId } });
```

Comment:
I might be missing something, but the tests name make it sound like we're looking for `created` === true but i'm not seeing that check.

Reply:
missed renaming test names in my refactor. Fixed!